make/common/MakeBase.gmk

```diff
@@ -127,14 +127,14 @@ endef
 # the build was created from
 SOURCE_REVISION_TRACKER := $(SUPPORT_OUTPUTDIR)/src-rev/source-revision-tracker
 
-# Locate all hg repositories included in the forest, as absolute paths
+# Locate all sourcecode repositories included in the forest, as absolute paths
 FindAllReposAbs = \
     $(strip $(sort $(dir $(filter-out $(TOPDIR)/build/%, $(wildcard \
         $(addprefix $(TOPDIR)/, .hg */.hg */*/.hg */*/*/.hg */*/*/*/.hg) \
```

Comment:
This line should be possible to remove as well.

Reply:
Hi Erik, thanks for the remark. I removed the line.